es/es-server/src/main/java/org/elasticsearch/Version.java

```diff
@@ -332,14 +332,21 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
     }
 
     /**
-     * Returns the minimum compatible version based on the current
-     * version. Ie a node needs to have at least the return version in order
-     * to communicate with a node running the current version. The returned version
-     * is in most of the cases the smallest major version release unless the current version
-     * is a beta or RC release then the version itself is returned.
+     * The minimum version other nodes need to have to communicate with this node.
+     *
+     * We support rolling upgrades between successive versions.
+     * So:
+     *
+     * <pre>
+     *  current | min support
+     *      4.1 |         4.0
+     *      4.2 |         4.1
+     *      ...
+     *      5.0 |         4.x where x is the last within 4.
+     * </pre>
      */
     public Version minimumCompatibilityVersion() {
-        return V_4_0_0;
+        return V_4_1_1;
```

Comment:
why not 4.1.0?

Reply:
4.1.0 never moved to the stable channels.  But can  also change it

Reply:
I'd suggest to either use `4.1.0` (I think even if never was stable it will be supported) or add a short comment to avoid confusion ;-)

Reply:
Changed it to `4.1.0`, in `crate-qa` the rolling upgrade tests start from there.